webextensions/api/bookmarks.json

```diff
@@ -227,7 +227,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks/getSubTree",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "14"
```

Comment:
✅ https://github.com/chromium/chromium/commit/532a8c69d659ee84c7b55ffd89d7af4a95c705c2 in [14.0.785.0](https://github.com/chromium/chromium/releases/tag/14.0.785.0) before 14.0.835